flink-java/pom.xml

```diff
@@ -64,6 +64,18 @@ under the License.
 			<version>0.5.1</version>
 		</dependency>
 
+		<dependency>
```

Comment:
This is pulling some unneeded dependencies: https://github.com/magro/kryo-serializers/blob/master/pom.xml
for example cglib,org.apache.wicket,